plugin/pkg/auth/authenticator/request/x509/x509.go

```diff
@@ -104,7 +105,13 @@ var DNSNameUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (
 
 // EmailAddressUserConversion builds user info from a certificate chain using the first EmailAddress on the certificate
 var EmailAddressUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (user.Info, bool, error) {
```

Comment:
Yeah, we're picking up emailAddress field of user certs as username in our deployment. However, the existing code doesn't seem to pick up this field correctly. Thought it might be useful to upstream this fix.